### ISSUE TITLE
Rails documentation is showing outdated docs 

### ISSUE THREAD

**Issue by jcoyne** [rails/rails-dom-testing#54]
http://edgeapi.rubyonrails.org/classes/ActionDispatch/Assertions/SelectorAssertions.html#method-i-assert_select

Shouldn't it look like this?

https://github.com/rails/rails-dom-testing/blob/master/lib/rails/dom/testing/assertions/selector_assertions.rb#L158-L160

**Comment by rafaelfranca** [comment]
Not sure what you mean, there are a lot of points in the documentation. Could you be more specific?

**Comment by rafaelfranca** [comment]
Oh. I see what you mean. That documentation is wrong. It is the Rails 4.1 documentation, how did you got that link? I tried to find it in the search in both api.rubyonrails.org and edgeapi.rubyonrails.org and could not.

@fxn should the edge api point to a file that doesn't exist in rails edge anymore?

**Comment by jcoyne** [comment]
I found a link on SO or a github issue when I noticed my tests were emitting deprecation warnings

**Comment by kaspth** [comment]
Strange the edgeapi still generates that. Maybe it's because the `ActionDispatch::Assertions::SelectorAssertions` class and underlying file was removed in either 4.2 or 5.0 (can't remember the deprecation cycle exactly) and then the documentation somehow wasn't aware of the doc file being supposed to be deleted.

**Comment by fxn** [comment]
Yeah, as of today edge docs are generated on top of the existing ones. If docs for *a.rb* were generated in the past and that file gets removed, the HTML remains orphan.

We'll do something about this.

**Comment by rafaelfranca** [comment]
@fxn did we fixed this?

**Comment by fxn** [comment]
@rafaelfranca Nope, I'll see what can be done... will write back.

**Comment by chancancode** [comment]
fwiw, I was looking for the docs the other day but couldn't find it. I realized that's because we moved the code out to a gem, so the docs gen couldn't/wouldn't find it. However, from what I understand the decision to move it out was purely for the benefit of code-organization/development, and these are still considered part of the main Rails API (it's a hard dependency of Rails), so it seems unfortunate that we lost the docs in the transition.
